src/app/tenders/compare-applicants/compare-applicants.component.ts

```diff
@@ -102,10 +104,13 @@ export class CompareApplicantsComponent implements OnInit {
       finInfoArr.push((typeof element.applicationFormDto?.financialInformation === 'string' ? JSON.parse(element.applicationFormDto?.financialInformation) : element.applicationFormDto?.financialInformation));
       compBankersArr.push((typeof element.applicationFormDto?.companyBankers === 'string' ? JSON.parse(element.applicationFormDto?.companyBankers) : element.applicationFormDto?.companyBankers));
       compAuditorsArr.push((typeof element.applicationFormDto?.companyAuditors === 'string' ? JSON.parse(element.applicationFormDto?.companyAuditors) : element.applicationFormDto?.companyAuditors));
-      if(element.tenderDetailsDto){
+      if (element.tenderDetailsDto) {
+        technicalDocArr.push(element.tenderDetailsDto?.contractorDocumentName);
```

Comment:
if condition is repeating multiple times use properly, technical document accordion and fields are still visible if data is not available.